.gitignore

```diff
@@ -92,3 +92,4 @@ typings/
 .next
 badge-examples.json
 supported-features.json
+service-definitions.yml
```

Comment:
If we're parsing it in the build/bundle step before its shipped to the browser I don't think its important. We might find that if others want to use it in future they prefer JSON, but for the moment lets think about our own use-case rather than trying to predict the needs of fictional users :)

Reply:
#776: I'm not making them up! 😉